### PR TITLE
Reviews have been added.

### PR DESCRIPTION
**Pull request by ruslanshakirov**

### CODE REVIEW

File: app/assets/javascripts/flash.coffee [hunk 1]
```diff
@@ -1,4 +1,4 @@
-$(document).ready ->
+$ ->
   alertBox = $(".alert-box")
   alertBox.fadeOut(4000)
   alertBox.hover (->
```
Comment (vladimirbazhanov) [inline app/assets/javascripts/flash.coffee]: this this stange mix of JS and Coffee, lets refactor this on monday

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  before_action :authenticate_user!
+  before_action :authorize_user!, only: %i(destroy update)
+  expose_decorated(:review, attributes: :review_params)
+  respond_to(:js)
+
+  def index
+  end
+
+  def create
+    review.story_id = params[:story_id]
```
Comment (vladimirbazhanov) [inline app/controllers/reviews_controller.rb]: you can merge story_id at review_params as you do it with user_id

File: app/controllers/stories_controller.rb [hunk 3]
```diff
@@ -30,14 +28,12 @@ def update
...
   end
 
-  private
-
-  def not_authorized
-    flash[:alert] = "You can't do it."
-    redirect_to(request.referrer || root_path)
+  def story_params
+    params.require(:story).permit(:name, :body, :genre, :full_description,
+      :short_description, :cover).merge(user_id: current_user.id)
```
Comment (vladimirbazhanov) [inline app/controllers/stories_controller.rb]: is this controller _always_ should set story for current_user?

File: app/policies/review_policy.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class ReviewPolicy < ApplicationPolicy
+  def reviewer_or_admin?
+    (record.user_id == user.id) || user.admin?
```
Comment (vladimirbazhanov) [inline app/policies/review_policy.rb]: record.user == user is the same, without ids

File: app/views/reviews/_review.html.slim [hunk 5]
```diff
@@ -0,0 +1,16 @@
+div id="review_#{review.id}"  
+  .custom-panel
+    .row
+      .medium-7.columns
+        h6 #{review.user_full_name}
```
Comment (vladimirbazhanov) [inline app/views/reviews/_review.html.slim]: h6 = review.user_full_name

#{} is better used for string interpolation only

**Comment by vladimirbazhanov** [comment]
👍 
we can talk about comment at monday

**Comment by ruslanshakirov** [comment]
Thank you, @vladimirbazhanov

File: app/controllers/stories_controller.rb (hunk 3 above)
Comment (ruslanshakirov) [inline app/controllers/stories_controller.rb]: No, when admin manages information of story, user_id was changed here.

**Pull request merged**